MCDataProducts/inc/CaloClusterNewMC.hh

```diff
@@ -0,0 +1,38 @@
+// This class described the CaloClusterNew MC truth. It contains the list of energy deposit from each 
```

Comment:
This was an intermediate commit, and the final name is CaloClusterMC.hh. As a rule, only intermediate commits have new, so that I can compare with the current data product before replacing them. So closing this one as well.